test/Microsoft.ML.FSharp.Tests/SmokeTests.fs

```diff
@@ -77,51 +74,23 @@ module SmokeTest1 =
     [<Fact>]
     let ``FSharp-Sentiment-Smoke-Test`` () =
 
-        // See https://github.com/dotnet/machinelearning/issues/401: forces the loading of ML.NET component assemblies
-        let _load  =
-            [ typeof<Microsoft.ML.Transforms.Text.TextNormalizingEstimator>; 
-              typeof<Microsoft.ML.Trainers.FastTree.FastTree>;
-              typeof<Microsoft.ML.EntryPoints.CVSplit>] // ML.EntryPoints
-
         let testDataPath = __SOURCE_DIRECTORY__ + @"/../data/wikipedia-detox-250-line-data.tsv"
 
-        let pipeline = Legacy.LearningPipeline()
-
-        pipeline.Add(
-             Microsoft.ML.Legacy.Data.TextLoader(testDataPath).CreateFrom<SentimentData>(
-                Arguments = 
-                    TextLoaderArguments(
-                        HasHeader = true,
-                        Column = [| TextLoaderColumn(Name = "Label", 
-                                                     Source = [| TextLoaderRange(0) |], 
-                                                     Type = Nullable (Legacy.Data.DataKind.Num))
-                                    TextLoaderColumn(Name = "SentimentText", 
-                                                     Source = [| TextLoaderRange(1) |], 
-                                                     Type = Nullable (Legacy.Data.DataKind.Text)) |] 
-                    )))
-
-        pipeline.Add(
-            TextFeaturizer(
-                "Features", [| "SentimentText" |],
-                KeepPunctuations = false,
-                OutputTokens = true,
-                VectorNormalizer = TextFeaturizingEstimatorTextNormKind.L2
-            ))
-
-        pipeline.Add(
-            FastTreeBinaryClassifier(
-                NumLeaves = 5, 
-                NumTrees = 5, 
-                MinDocumentsInLeafs = 2
-            ))
-
-        let model = pipeline.Train<SentimentData, SentimentPrediction>()
+        let ml = MLContext(seed = new System.Nullable<int>(1), conc = 1)
+        let data = ml.Data.ReadFromTextFile<SentimentData>(testDataPath, hasHeader = true)
+
+        let pipeline = ml.Transforms.Text.FeaturizeText("SentimentText", "Features") 
```

Comment:
By default it is set to 10. I am not sure how to create an Action<Arguments> object in F#, and I believe this will not change the code coverage. We have tests that are specific to FastTree for binary classification that set the MinDocumentsInLeafs to something different.